packages/shared/src/components/sidebar/ManageSection.tsx

```diff
@@ -25,52 +31,85 @@ export function ManageSection({
   onShowSettings,
   ...props
 }: ManageSectionProps): ReactElement {
+  const seoSidebar = useFeature(feature.seoSidebar);
+  const isV1Sidebar = seoSidebar === SeoSidebarExperiment.V1;
+  const { modal, openModal } = useLazyModal();
   const shouldShowDnD = !!process.env.TARGET_BROWSER;
```

Comment:
This should also depend on the sidebar experiement

Reply:
Yes, I think the evaluations are down below.